### PR TITLE
New API Path added "datawarehousequeries"  in sqlpool

### PR DESCRIPTION
**Pull request by 12VamshiKrishna**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [x] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [x] SDK of Js
      - [x] SDK of Go
      - [x] PowerShell
      - [x] CLI
      - [x] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following appy to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json [hunk 1]
```diff
@@ -7623,6 +7681,112 @@
...
+          "description": " QueryText Message",
+          "type": "string",
+          "readOnly": true
+        },
+        "importance": {
+          "description": "dataWarehouseQueries Importance",
+          "type": "string",
+          "readOnly": true
+        },
+        "groupName": {
```
Comment (bansalaseem) [inline specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json:7743]: Can you please share what is group name? Is it an enum type or something else?

File: specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json [hunk 2]
```diff
@@ -7623,6 +7681,112 @@
...
+          "description": "Requested GroupName ",
+          "type": "string",
+          "readOnly": true
+        },
+        "classifierName": {
```
Comment (bansalaseem) [inline specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json:7748]: Can you please share what is classifier name? Is it an enum type or something else?

File: specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json [hunk 3]
```diff
@@ -7623,6 +7681,112 @@
...
+          "description": "AllocationPercentage of the Resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "error": {
+          "description": "The operation error message.",
+          "type": "string",
+          "readOnly": true
+        },
+        "login": {
```
Comment (bansalaseem) [inline specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json:7763]: Does this contain user names or something else? I am asking this to see whether it is an enum or not

File: specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json [hunk 4]
```diff
@@ -7623,6 +7681,112 @@
...
+          "type": "number",
+          "format": "int32",
+          "readOnly": true
+        },
+        "sessionId": {
+          "description": "SessionId for Resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceClass": {
```
Comment (bansalaseem) [inline specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json:7779]: What are the potential values of resource class?

File: specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json [hunk 5]
```diff
@@ -7623,6 +7681,112 @@
...
+          "description": "Requested ResourceClass",
+          "type": "string",
+          "readOnly": true
+        },
+        "concurrencySlots": {
```
Comment (bansalaseem) [inline specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json:7784]: what are the potential values of concurrency slots?

Comment (12VamshiKrishna) [reply specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json:7784]: it is a integer value 0-1200 as a string or "N/A"

File: specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json (hunk 3 above)
Comment (12VamshiKrishna) [reply specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json:7763]: @bansalaseem login is the login name of the user

File: specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json (hunk 2 above)
Comment (12VamshiKrishna) [reply specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json:7748]: @bansalaseem It is not Enum, Customer creates Classifier, so Names are custom

File: specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json (hunk 1 above)
Comment (12VamshiKrishna) [reply specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json:7743]: @bansalaseem It is not Enum, Customer creates Group, so Names are custom

File: specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json (hunk 4 above)
Comment (12VamshiKrishna) [reply specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json:7779]: @bansalaseem added values for resource class

**Review by bansalaseem — APPROVED** [review]

**Pull request merged**